assets/src/scripts/components/Cards/SearchForm.tsx

```diff
@@ -48,6 +46,8 @@ export default function SearchForm() {
     SEARCH_OPTIONS.term.value,
   );
 
+  if (!codingSystemName || !searchURL) return null;
+
```

Comment:
I have no issue with this, but it's not immediately obvious to me what difference the change to the position of this line makes.